e2e-tests/production-runtime/cypress/integration/compilation-hash.js

```diff
@@ -60,7 +60,7 @@ describe(
 
         cy.visit(`/`).waitForRouteChange()
         let didMock = false
-        cy.intercept("/app-data.json", req => {
+        cy.intercept("/page-data/app-data.json", req => {
```

Comment:
https://docs.cypress.io/guides/references/migration-guide#URL-matching-is-stricter